File: Trabalho LP 3/e1/java/src/Main.java
import java.util.Scanner;

public class Main {

    static boolean verificar_nota(float nota) {

        return nota >= 7;
    }
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        float[] nota_alunos = new float[10];
        boolean resultado;

        System.out.println("======== AVALIACAO 3 ========");
        System.out.println("=========== IFBA ============\n");
        System.out.println("OBS. INSIRA OS VALORES COMO NO EXEMPLO: 10,0\n");

        for(int i = 0; i <=9; i++){
            System.out.printf("NOTA ALUNO %d: ", i+1);
            nota_alunos[i] = ler.nextFloat();
        }

        System.out.println("=============================\n");

        for(int i = 0; i <= 9; i++){
            resultado = verificar_nota(nota_alunos[i]);

            if(resultado){
                System.out.printf("ALUNO %d foi APROVADO em linguagem de programacao.", i+1);
            }else{
                System.out.printf("ALUNO %d foi REPROVADO em linguagem de programacao.", i+1);
            }

            System.out.println("\n");
        }
    }
}

File: Trabalho LP 3/e1/python/main.py
def verificar_nota(nota, x):
    #aprovado = 1 reprovado = 0

    if nota[x] >= 7:
        return True
    else:
        return False

nota_alunos = []

print ('======== AVALIACAO 3 ========')
print ('=========== IFBA ============\n')

for i in range (10):
    aluno_atual = i + 1
    pergunta = 'NOTA ALUNO ' + str(aluno_atual) + ': '
    nota_atual = float(input(pergunta))
    nota_alunos.insert(i, nota_atual)

print ('=============================\n')

for x in range (10):
    resultado = verificar_nota(nota_alunos, int(x))
    aluno_agora = int(x + 1)

    if resultado == True:
        print ('ALUNO', aluno_agora,' foi APROVADO em linguagem de programacao.')


    if resultado == False:
       print ('ALUNO', aluno_agora,'foi REPROVADO em linguagem de programacao.')